deploy/docker/seqr/deploy_cloudrun_jobs.sh

```diff
@@ -0,0 +1,107 @@
+# Add updater scripts
+
+ENV_NAME="prod"
+
+SEQR_URL="https://seqr.populationgenomics.org.au/"
+SERVICE_ACCOUNT="seqr-prod@seqr-308602.iam.gserviceaccount.com"
+IMAGE="australia-southeast1-docker.pkg.dev/seqr-308602/seqr-project/seqr:gcloud-prod"
+
+POSTGRES_USERNAME="postgres"
+POSTGRES_DATABASE="seqrdb"
+POSTGRES_REFERENCE_DATABASE="reference_data_db"
+POSTGRES_PASSWORD_SECRET_NAME="postgres-password"
+
+GSA_SECRET_NAME="gsa-key-prod"
+DJANGO_SECRET_NAME="django-key-prod"
+
+gcloud beta run jobs create \
```

Comment:
Not super important, but given that only the `args` seem to differ below, a `for` loop could avoid a lot of the code duplication.

Reply:
I thought about it, the main difference is the reference updater uses more memory - not planning to use it a lot (more for reference) so for now I'll just leave it.